3.1.0 fails to find `drm.h`

Hello,

I am using Gentoo and would like to package 3.1.0 for the GURU repos, where 3.0.2 work fine.
However during compilation I get those errors:
```log
[12/21] /usr/bin/x86_64-pc-linux-gnu-gcc -DHAS_REALLOCARRAY -DUSING_LIBUDEV -D_GNU_SOURCE -I/var/tmp/portage/sys-process/nvtop-3.1.0/work
/nvtop-3.1.0/include -I/var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0_build/include  -march=native -O2 -pipe  -flto=8 -Werror=
odr -Werror=lto-type-mismatch -Werror=strict-aliasing -std=gnu11 -MD -MT src/CMakeFiles/nvtop.dir/extract_gpuinfo_panthor.c.o -MF src/CMa
keFiles/nvtop.dir/extract_gpuinfo_panthor.c.o.d -o src/CMakeFiles/nvtop.dir/extract_gpuinfo_panthor.c.o -c /var/tmp/portage/sys-process/n
vtop-3.1.0/work/nvtop-3.1.0/src/extract_gpuinfo_panthor.c
FAILED: src/CMakeFiles/nvtop.dir/extract_gpuinfo_panthor.c.o 
/usr/bin/x86_64-pc-linux-gnu-gcc -DHAS_REALLOCARRAY -DUSING_LIBUDEV -D_GNU_SOURCE -I/var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0/include -I/var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0_build/include  -march=native -O2 -pipe  -flto=8 -Werror=odr -Werror=lto-type-mismatch -Werror=strict-aliasing -std=gnu11 -MD -MT src/CMakeFiles/nvtop.dir/extract_gpuinfo_panthor.c.o -MF src/CMakeFiles/nvtop.dir/extract_gpuinfo_panthor.c.o.d -o src/CMakeFiles/nvtop.dir/extract_gpuinfo_panthor.c.o -c /var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0/src/extract_gpuinfo_panthor.c
In file included from /var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0/src/extract_gpuinfo_panthor.c:21:
/usr/include/xf86drm.h:40:10: fatal error: drm.h: No such file or directory
   40 | #include <drm.h>
      |          ^~~~~~~
compilation terminated.
[13/21] /usr/bin/x86_64-pc-linux-gnu-gcc -DHAS_REALLOCARRAY -DUSING_LIBUDEV -D_GNU_SOURCE -I/var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0/include -I/var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0_build/include  -march=native -O2 -pipe  -flto=8 -Werror=odr -Werror=lto-type-mismatch -Werror=strict-aliasing -std=gnu11 -MD -MT src/CMakeFiles/nvtop.dir/extract_gpuinfo_panfrost.c.o -MF src/CMakeFiles/nvtop.dir/extract_gpuinfo_panfrost.c.o.d -o src/CMakeFiles/nvtop.dir/extract_gpuinfo_panfrost.c.o -c /var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0/src/extract_gpuinfo_panfrost.c
FAILED: src/CMakeFiles/nvtop.dir/extract_gpuinfo_panfrost.c.o 
/usr/bin/x86_64-pc-linux-gnu-gcc -DHAS_REALLOCARRAY -DUSING_LIBUDEV -D_GNU_SOURCE -I/var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0/include -I/var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0_build/include  -march=native -O2 -pipe  -flto=8 -Werror=odr -Werror=lto-type-mismatch -Werror=strict-aliasing -std=gnu11 -MD -MT src/CMakeFiles/nvtop.dir/extract_gpuinfo_panfrost.c.o -MF src/CMakeFiles/nvtop.dir/extract_gpuinfo_panfrost.c.o.d -o src/CMakeFiles/nvtop.dir/extract_gpuinfo_panfrost.c.o -c /var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0/src/extract_gpuinfo_panfrost.c
In file included from /var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0/src/extract_gpuinfo_panfrost.c:24:
/usr/include/xf86drm.h:40:10: fatal error: drm.h: No such file or directory
   40 | #include <drm.h>
      |          ^~~~~~~
compilation terminated.
```
Here's what DRM headers I do have:
```console
$ find /usr/include/ -name "drm.h"
/usr/include/drm/drm.h
/usr/include/libdrm/drm.h
/usr/include/wlroots-0.18/wlr/backend/drm.h
```
I am using libdrm-2.4.123-r1

After manually changing `xf86drm.h` to include not `drm.h` but `drm/drm.h` or `libdrm/drm.h` I get this:
```
[16/21] /usr/bin/x86_64-pc-linux-gnu-gcc -DHAS_REALLOCARRAY -DUSING_LIBUDEV -D_GNU_SOURCE -I/var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0/include -I/var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0_build/include  -march=native -O2 -pipe  -flto=8 -Werror=odr -Werror=lto-type-mismatch -Werror=strict-aliasing -std=gnu11 -MD -MT src/CMakeFiles/nvtop.dir/extract_gpuinfo_panthor.c.o -MF src/CMakeFiles/nvtop.dir/extract_gpuinfo_panthor.c.o.d -o src/CMakeFiles/nvtop.dir/extract_gpuinfo_panthor.c.o -c /var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0/src/extract_gpuinfo_panthor.c
FAILED: src/CMakeFiles/nvtop.dir/extract_gpuinfo_panthor.c.o 
/usr/bin/x86_64-pc-linux-gnu-gcc -DHAS_REALLOCARRAY -DUSING_LIBUDEV -D_GNU_SOURCE -I/var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0/include -I/var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0_build/include  -march=native -O2 -pipe  -flto=8 -Werror=odr -Werror=lto-type-mismatch -Werror=strict-aliasing -std=gnu11 -MD -MT src/CMakeFiles/nvtop.dir/extract_gpuinfo_panthor.c.o -MF src/CMakeFiles/nvtop.dir/extract_gpuinfo_panthor.c.o.d -o src/CMakeFiles/nvtop.dir/extract_gpuinfo_panthor.c.o -c /var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0/src/extract_gpuinfo_panthor.c
In file included from /var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0/src/extract_gpuinfo_panthor.c:28:
/var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0/src/panthor_drm.h:6:10: fatal error: drm.h: No such file or directory
    6 | #include "drm.h"
      |          ^~~~~~~
compilation terminated.
[17/21] /usr/bin/x86_64-pc-linux-gnu-gcc -DHAS_REALLOCARRAY -DUSING_LIBUDEV -D_GNU_SOURCE -I/var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0/include -I/var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0_build/include  -march=native -O2 -pipe  -flto=8 -Werror=odr -Werror=lto-type-mismatch -Werror=strict-aliasing -std=gnu11 -MD -MT src/CMakeFiles/nvtop.dir/extract_gpuinfo_panfrost.c.o -MF src/CMakeFiles/nvtop.dir/extract_gpuinfo_panfrost.c.o.d -o src/CMakeFiles/nvtop.dir/extract_gpuinfo_panfrost.c.o -c /var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0/src/extract_gpuinfo_panfrost.c
FAILED: src/CMakeFiles/nvtop.dir/extract_gpuinfo_panfrost.c.o 
/usr/bin/x86_64-pc-linux-gnu-gcc -DHAS_REALLOCARRAY -DUSING_LIBUDEV -D_GNU_SOURCE -I/var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0/include -I/var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0_build/include  -march=native -O2 -pipe  -flto=8 -Werror=odr -Werror=lto-type-mismatch -Werror=strict-aliasing -std=gnu11 -MD -MT src/CMakeFiles/nvtop.dir/extract_gpuinfo_panfrost.c.o -MF src/CMakeFiles/nvtop.dir/extract_gpuinfo_panfrost.c.o.d -o src/CMakeFiles/nvtop.dir/extract_gpuinfo_panfrost.c.o -c /var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0/src/extract_gpuinfo_panfrost.c
In file included from /var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0/src/extract_gpuinfo_panfrost.c:31:
/var/tmp/portage/sys-process/nvtop-3.1.0/work/nvtop-3.1.0/src/panfrost_drm.h:9:10: fatal error: drm.h: No such file or directory
    9 | #include "drm.h"
      |          ^~~~~~~
compilation terminated.
```

From what I can tell there is no `drm.h` shipped with 3.1.0, or at https://github.com/Syllo/nvtop/commit/6e91c745cd051d902fc57e9195068df03eb192bb.

I have checked and `panthor_drm.h` and `panfrost_drm.h` want to include a local `drm.h` that is not there.

If I edit them to include `<libdrm/drm.h>` instead of `"drm.h"` it compiles fine.

Here are my questions:
1. Is there a problem with packaging libdrm on Gentoo (`xf86drm.h`)?
2. What should provide `pan{thor,frost}_drm.h`?

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -74,7 +74,7 @@ if(AMDGPU_SUPPORT OR INTEL_SUPPORT)
   target_sources(nvtop PRIVATE device_discovery_linux.c)
 endif()
 
-if(AMDGPU_SUPPORT OR MSM_SUPPORT)
+if(AMDGPU_SUPPORT OR MSM_SUPPORT OR PANFROST_SUPPORT OR PANTHOR_SUPPORT)
   # Search for libdrm for AMDGPU support
   find_package(Libdrm)
 

